relays/bin-substrate/src/chains/wococo.rs

```diff
@@ -18,13 +18,14 @@
 
 use crate::cli::CliChain;
 use relay_bridge_hub_wococo_client::BridgeHubWococo;
+use relay_substrate_client::SimpleRuntimeVersion;
 use relay_wococo_client::Wococo;
-use sp_version::RuntimeVersion;
```

Comment:
@serban300 
I guess, we can also remove dependency `sp-version` at least from `bin-substrate` and maybe from other modules,
could you check it please?

Reply:
You're right. Opened #1758 for this.